client/src/pages/SchedulerCalendar.js

```diff
@@ -0,0 +1,77 @@
+import React, { useEffect, useState } from "react";
+import moment from "moment";
+import SchedulerWidget from "../component/SchedularWidget";
+const SchedulerCalendar = (props) => {
+  const [freeSlots, setFreeSlots] = useState([]);
+  const [dateSelected, setDateSelected] = useState(new Date());
+  const meetingTime = props.match.params.meetingTime;
+  useEffect(() => {
+    let access_token = localStorage.getItem("access_token");
+    let email = localStorage.getItem("email");
```

Comment:
we should be using props, or if props have become too much of a hassle, a user Context or something. localstorage should really only be used for auth or very basic caching

Reply:
yes. Context is coming to solve all issues related to local storage